src/pybind/mgr/dashboard_v2/tests/test_auth.py

```diff
@@ -12,6 +12,7 @@
 from ..tools import Session
 
 
+@skipIf(bool(os.environ.get('SKIP_LOADED_BY_MGR', False)), 'Needs to run in mgr')
```

Comment:
This code is duplicated many times in this PR, so it seems to me a good candidate for a new decorator.